addons/website/static/src/builder/plugins/options/customers_avatars_option_plugin.js

```diff
@@ -0,0 +1,97 @@
+import { Plugin } from "@html_editor/plugin";
+import { registry } from "@web/core/registry";
+import { BuilderAction } from "@html_builder/core/builder_action";
+class AvatarCountAction extends BuilderAction {
+    static id = "avatarCount";
+
+    getValue({ editingElement, params: { mainParam: action } = {} }) {
+        if (action === 'countValue') {
+            return editingElement.dataset.countValue || '10+';
+        }
+        if (action === 'showCount') {
+            return editingElement.classList.contains('o_show_count');
+        }
+        return null;
+    }
+
+    isApplied({ editingElement, params: { mainParam: action } = {} }) {
+        if (action === 'showCount') {
+            const countElement = editingElement.querySelector('.o_avatar_count');
+            return countElement && !countElement.classList.contains('d-none');
+        }
+        return false;
+    }
+
+    apply({ editingElement, params: { mainParam: action } = {}, value }) {
+        if (action === 'showCount') {
+            const countElement = this._ensureCountElement(editingElement);
+            if (countElement) {
+                countElement.classList.toggle('d-none');
+                countElement.classList.toggle('d-flex');
```

Comment:
Never `toggle` without the second param. You receive if you have to toggle on or off via the `value` parameter: use that. DOM manipulation via options should never rely on current DOM: only on values sent by the builder.